personal/privacyidea/class_mfaAccount.inc

```diff
@@ -192,11 +192,70 @@ class mfaAccount extends plugin
         /* Call parent execute */
         plugin::execute();
 
-        // Always mark this as an account.
         if (!$this->is_account && $this->acl_is_writeable("")) {
             $this->is_account = true;
         }
 
+        // Act on account stat changes
+        $display = "";
+        if ($this->parent instanceof roletabs || $this->parent instanceof uniBonnAccountTabs) {
+            if (isset($_POST['modify_state'])) {
+                if ($this->is_account && $this->acl_is_removeable()) {
+                    $this->is_account = false;
+                } elseif (!$this->is_account && $this->acl_is_createable()) {
+                    $obj = $this->parent->by_object['accountAuth'];
```

Comment:
wrong class name, but will be removed anyway...

Reply:
See above.